airflow/cli/cli_parser.py

```diff
@@ -66,6 +66,17 @@ def _check_value(self, action, value):
         if value == 'celery' and executor != ExecutorLoader.CELERY_EXECUTOR:
             message = f'celery subcommand works only with CeleryExecutor, your current executor: {executor}'
             raise ArgumentError(action, message)
+        if value == 'kubernetes':
+            try:
+                from kubernetes.client import models
+                if not models:
+                    message = 'kubernetes subcommand requires that ' \
```

Comment:
Can you add full pip install command - ``pip install 'apache-airflow[cncf.kubernetes]' ``?